dashboard/db/migrate/20151105100000_create_site_metrics.rb

```diff
@@ -0,0 +1,18 @@
+class CreateSiteMetrics < ActiveRecord::Migration
+  def change
+    create_table(:site_metrics, id: false) do |t|
+      t.date 'date'
```

Comment:
Does this table need an index on date?  It seems like we'll often want it scan date ranges.